Mathlib/Order/RelIso/Basic.lean

```diff
@@ -393,32 +393,80 @@ protected theorem isWellOrder : ∀ (_ : r ↪r s) [IsWellOrder β s], IsWellOrd
   | f, H => { f.isStrictTotalOrder with wf := f.wellFounded H.wf }
 #align rel_embedding.is_well_order RelEmbedding.isWellOrder
 
+end RelEmbedding
```

Comment:
Where did this come from? I don't see it in the mathport diff.

Reply:
I added it to avoid writing `_root_` a bunch of times.

Reply:
I think it's best to just port faithfully; the more you refactor when porting the harder it becomes to forward-port subsequent changes.